text/0000-signed-transaction-payload/0000-signed-transaction-payload.md

```diff
@@ -0,0 +1,825 @@
+                                                    <tr>
+                                                        <td>Address</td>
+                                                        <td>ByteArray[49]</td>
+                                                        <td>The T5B1 encoded WOTS address.</td>
+                                                    </tr>
+                                                </table>
+                                            </details>
+                                            <details>
+                                                <summary>Ed25519 Address</summary>
+                                                <table>
+                                                    <tr>
+                                                        <td><b>Name</b></td>
+                                                        <td><b>Type</b></td>
+                                                        <td><b>Description</b></td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address Type</td>
+                                                        <td>varint</td>
+                                                        <td>
+                                                            Set to <strong>value 0</strong> to denote an <i>Ed25519 Address</i>.
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address</td>
+                                                        <td>ByteArray[32]</td>
+                                                        <td>The raw bytes of the Ed25519 address which is a BLAKE2b-256 hash of the Ed25519 public key.</td>
+                                                    </tr>
+                                                </table>
+                                            </details>
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>Amount</td>
+                                        <td>uint64</td>
+                                        <td>The amount of tokens to deposit with this <i>SigLockedSingleDeposit</i> output.</td>
+                                    </tr>
+                                </table>
+                            </details>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Payload Length</td>
+                        <td>varint</td>
+                        <td>The length in bytes of the optional payload.</td>
+                    </tr>
+                   <tr>
+                        <td valign="top">Payload <code>optOneOf</code></td>
```

Comment:
Can we support multiple payloads? For example, if one wants to pass a tag and also some unsigned data?

Reply:
The general idea would be to have a new payload type `PayloadChain` or something that supports something like this. However, in order to keep the very initial payload types to a minimum it has not been spec'ed yet.

Reply:
@Wollac Sounds good.